liquidctl/driver/msi_7zw0led.py

```diff
@@ -0,0 +1,43 @@
+"""liquidctl drivers for MSI liquid coolers.
+
+Supported devices:
+
+- MPG CORELIQUID K360
+
+Copyright (C) 2020–2021  Andrew Udvare and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+from liquidctl.driver.usb import UsbHidDriver
+
+_REPORT_LENGTH = 64
+
+
+class MSI7ZW0LED(UsbHidDriver):
```

Comment:
`MSI7ZW0LED` is quite hard to read.  I think `MpgCooler` is a better one.

("MPG" is important because the "MAG" line appears to use a different OEM, meaning it will probably also use a different protocol; on the other hand, "MSI" can be inferred from "MPG").